pywbemtools/pywbemcli/_cmd_connection.py

```diff
@@ -187,9 +188,15 @@ def connection_save(context, name, **options):
     """
     Save the current connection to a new WBEM connection definition.
 
-    Create a new WBEM connection definition in the connections file from the
-    current connection. A connection definition with the name NAME must not yet
-    exist. The NAME argument (the name of the new connection) is required.
+    Save WBEM connection definition in the connections file from the current
+    connection. The NAME argument (the name of the new connection) is required.
+    If the connection exists, it will be overwritten with the current
+    connection definition.
```

Comment:
I think it would be good to be more clear as to what is considered an existing connection. I suggest to change "If the connection exists" to "If a connection definition with that name already exists".

Reply:
DONE